netlify.toml

```diff
@@ -6,4 +6,4 @@
 # Deploy Preview context: All Deploy Previews
 # will inherit these settings.
 [context.deploy-preview]
-  command = "jekyll build -D --future && cp _redirects _site/"
+  command = "jekyll build --trace -D --future && cp _redirects _site/"
```

Comment:
Debug setting?

Reply:
It is a debug setting, but no reason to get rid of it. Will help with future debugging issues if we have them.